### PR TITLE
Fix Safari 3 versions that were in Safari 1

### PR DESCRIPTION
**Pull request by foolip**
Follow up to https://github.com/mdn/browser-compat-data/pull/16509.

### CODE REVIEW

File: api/HTMLAnchorElement.json [hunk 1]
```diff
@@ -78,7 +78,7 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "3"
+              "version_added": "1"
```
Comment (foolip) [inline api/HTMLAnchorElement.json:81]: https://github.com/WebKit/WebKit/blob/a063c2b75ddebd1edfe9c54c428d762bdb61f794/WebCore/khtml/ecma/kjs_html.cpp#L620

File: api/HTMLAnchorElement.json [hunk 2]
```diff
@@ -127,7 +127,7 @@
...
```
Comment (foolip) [inline api/HTMLAnchorElement.json:130]: https://github.com/WebKit/WebKit/blob/a063c2b75ddebd1edfe9c54c428d762bdb61f794/WebCore/khtml/ecma/kjs_html.cpp#L621

File: api/HTMLAnchorElement.json [hunk 3]
```diff
@@ -274,7 +274,7 @@
...
```
Comment (foolip) [inline api/HTMLAnchorElement.json:277]: https://github.com/WebKit/WebKit/blob/a063c2b75ddebd1edfe9c54c428d762bdb61f794/WebCore/khtml/ecma/kjs_html.cpp#L638

File: api/HTMLAnchorElement.json [hunk 4]
```diff
@@ -518,7 +518,7 @@
...
```
Comment (foolip) [inline api/HTMLAnchorElement.json:521]: https://github.com/WebKit/WebKit/blob/a063c2b75ddebd1edfe9c54c428d762bdb61f794/WebCore/khtml/ecma/kjs_html.cpp#L640

File: api/HTMLAnchorElement.json [hunk 5]
```diff
@@ -567,7 +567,7 @@
...
```
Comment (foolip) [inline api/HTMLAnchorElement.json:570]: https://github.com/WebKit/WebKit/blob/a063c2b75ddebd1edfe9c54c428d762bdb61f794/WebCore/khtml/ecma/kjs_html.cpp#L641

File: api/HTMLLinkElement.json [hunk 6]
```diff
@@ -331,7 +331,7 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "3"
+              "version_added": "1"
```
Comment (foolip) [inline api/HTMLLinkElement.json:334]: https://github.com/WebKit/WebKit/blob/a063c2b75ddebd1edfe9c54c428d762bdb61f794/WebCore/khtml/ecma/kjs_html.cpp#L307

File: api/HTMLStyleElement.json [hunk 7]
```diff
@@ -127,7 +127,7 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "3"
+              "version_added": "1"
```
Comment (foolip) [inline api/HTMLStyleElement.json:130]: https://github.com/WebKit/WebKit/blob/a063c2b75ddebd1edfe9c54c428d762bdb61f794/WebCore/khtml/ecma/kjs_html.cpp#L345

**Review by queengooborg — APPROVED** [review]

**Pull request merged**